src/pf-notification-drawer/pf-notification-drawer.scss

```diff
@@ -0,0 +1,46 @@
+.drawer-pf {
```

Comment:
In general, we should be using CSS Patternfly from Patterfly. 

For the Angular Patternfly repo; for example, we consume the Patternfly CSS. If end up creating custom CSS in the repo, that is typically a red flag and should be added to Patternfly instead. That way, the CSS can be consumed by other repos; Angular Patternfly, patternfly-ng, etc.